sycl/doc/extensions/proposed/sycl_ext_intel_user_defined_reductions.asciidoc

```diff
@@ -0,0 +1,138 @@
+= sycl_ext_intel_user_defined_reductions
```

Comment:
```suggestion
= sycl_ext_oneapi_user_defined_reductions
```
I don't think anything in here is Intel-specific.

Reply:
Applied in 26b93170f6afa6dca47f62bf487b65e5854e0c54